### PR TITLE
feat-structure

### PR DESCRIPTION
**Pull request by AmishMishra11**
Created  home page for the E-commerce website 
Contents
->Navigation Bar
->Category
->Offers
->Main Content
->New Arrivals

https://user-images.githubusercontent.com/76205249/153034186-a1999534-303c-4a39-9188-d4d3c862a924.mp4

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems good...
Just one suggestion try to remove the commented html..

**Comment by AmishMishra11** [comment]
> Code seems good... Just one suggestion try to remove the commented html..

Thank you for pointing it out, I've updated the HTML

File: index.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+      rel="stylesheet"
+      href="https://am-component.netlify.app/stylesheets/components.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+
+    <title>Mercart Home Page</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="container-nav">
```
Comment (Tarandeep-s1ngh) [inline index.html:19]: Try to use semantic html

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (AmishMishra11) [reply index.html:19]: > Try to use semantic html

Thanks, will implement them

**Pull request merged**